suites/squid/rgw/tier-2_rgw_ssl_cephadm_gen_cert.yaml

```diff
@@ -302,18 +300,6 @@ tests:
             verify-io-on-site: ["ceph-pri"]
             config-file-name: test_Mbuckets_with_Nobjects_compression.yaml
 
-  - test:
```

Comment:
@anrao19 These tests may be duplicates but added to test with cephadm rgw ssl gen_cert

Reply:
sure @hmaheswa